### PR TITLE
TS Loader module

### PR DESCRIPTION
**Pull request by DamnClin**
Small module to add a Loader object for frontend applications

### CODE REVIEW

File: src/main/java/tech/jhipster/lite/generator/client/loader/infrastructure/primary/TsLoaderModuleConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+@Configuration
+class TsLoaderModuleConfiguration {
+
+  @Bean
+  JHipsterModuleResource tsLoaderModule(TsLoaderModuleApplicationService tsLoader) {
+    return JHipsterModuleResource.builder()
+      .slug(JHLiteModuleSlug.TS_LOADER)
+      .withoutProperties()
+      .apiDoc("Client", "Helper class to represent loading states")
+      .organization(JHipsterModuleOrganization.builder().addDependency(JHLiteModuleSlug.PRETTIER).build())
```
Comment (murdos) [inline src/main/java/tech/jhipster/lite/generator/client/loader/infrastructure/primary/TsLoaderModuleConfiguration.java:19]: Is there a reason for requiring the prettier module?

Comment (DamnClin) [reply src/main/java/tech/jhipster/lite/generator/client/loader/infrastructure/primary/TsLoaderModuleConfiguration.java:19]: Not really, was hesitating... Init is probably a better idea

**Review by murdos — APPROVED** [review]

File: src/main/java/tech/jhipster/lite/generator/client/loader/infrastructure/primary/TsLoaderModuleConfiguration.java (hunk 1 above)
Comment (murdos) [reply src/main/java/tech/jhipster/lite/generator/client/loader/infrastructure/primary/TsLoaderModuleConfiguration.java]: In fact this requires typescript, but our client modules and the typescript module don't have any thing in common, so we would have to choose between client or typescript as prerequisite.
Maybe we should try to have some (all?) of your client modules depend on the typescript module (and remove the typscript setup they embed). WDYT?

Comment (DamnClin) [reply src/main/java/tech/jhipster/lite/generator/client/loader/infrastructure/primary/TsLoaderModuleConfiguration.java]: Hopefully I'll find some motivation to work on the frontend modules in the next weeks and I'll probably stumble into that problem...

**Pull request merged**